### ISSUE TITLE
The HTML <Title> element is not useful for navigation

### ISSUE THREAD

**Issue by auspex** [ledgersmb/LedgerSMB#8903]
### Version

1.12.9

### What browsers are you seeing the problem on?

Chrome, This problem isn't browser related

### What happened?

On the login page, the <title> element is:
```
<title>LedgerSMB 1.12.9</title>
```
Once logged in, my title becomes:
```
<title>LedgerSMB 1.12.9 -- derek -- coastalvoices</title>
```
on **every** page. This is not useful for those of us who routinely use the back button to navigate.

### What should have happened?

I expect the HTML Title to reflect _where_  I am within a website.

The current title is a lot of text and most of it not useful. 

If you're the site admin, you really should know what version of LSMB you're running, and users don't _care_. Most websites would put that info in a page footer or an 'About' page.

Similarly, I probably know who I'm logged in as. Most people won't log in under more than one name, an admin like me might log in as 'admin' and under my own name, but rarely anything else. If you really need to check who you are, it should be in the preferences page (most websites would have a User icon where those would be set).

Now, it may well be a good idea to put the database name there — I suspect a lot of people using LSMB are bookkeeping for multiple companies — but I suggest a much better use of the title tag would be:
```
<title>LedgerSMB—{DB NAME}—{MENU ENTRY}</title>
```

**Comment by ehuelsmann** [comment]
Ok. the above patch is based on old documentation. `this.$router.push()` returns a promise on Vue-Router 4.x and doesn't take an `onComplete` argument anymore.
